ceph-dev-new-build/build/setup_rpm

```diff
@@ -44,10 +44,6 @@ pwd
 
 setup_rpm_build_deps
 
-if [[ $CI_CONTAINER == "true" && $DISTRO == "centos" && "$RELEASE" =~ 8|9 ]] ; then
```

Comment:
The hope of having this section here was to fail the build early if the credentials were set incorrectly (granted, there might have been a comment to that effect, sorry).  Are you thinking that being able to run container builds separately from previously-built packages will be seamless enough that this isn't necessary anymore?

Reply:
Ah, no that wasn't clear. But yeah, since this pipeline does support building containers from shaman repos, I'm thinking it's not as important to check the credentials early anymore. They don't break regularly, right...?